device/delta/x86_64-delta_ag9032v1-r0/led_proc_init.soc

```diff
@@ -1,3 +1,1027 @@
+phy ce0 AMS_TX_CTL2r.0  AMS_TX_AMP_CTL=0xf
```

Comment:
can these pre-emphasis settings be put into config.bcm?

Reply:
Hi lguohan, I will put pre-emphasis into config.bcm.